### PR TITLE
Add Browser Extensions and Apps

### PR DESCRIPTION
**Pull request by vitalets**
**https://github.com/vitalets/awesome-browser-extensions-and-apps**

**It is the list of resources useful for developing browser extensions and apps. I suggest separate list because extensions development has many special topics such as working with browser api, packaging and deploying to store.**

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [ ] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [ ] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

Thanks in advance!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -184,6 +184,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Progressive Web Apps](https://github.com/TalAter/awesome-progressive-web-apps)
 - [choo](https://github.com/YerkoPalma/awesome-choo) - App framework.
 - [Redux](https://github.com/brillout/awesome-redux) - State container for JavaScript apps.
+- [Awesome Browser Extensions and Apps](https://github.com/vitalets/awesome-browser-extensions-and-apps)
```
Comment (vdemedes) [inline readme.md]: I believe you should remove "Awesome " prefix, as all of these lists are "awesome" :)

Comment (vitalets) [inline readme.md]: oh, thanks! Sure.

**Comment by sindresorhus** [comment]
Good start, but the list is not very comprehensive. Awesome lists are hard work. Put some more work into it. Take a look at some other lists and see how they do it. E.g. https://github.com/sindresorhus/awesome-electron https://github.com/sindresorhus/awesome-npm Try resubmitting in a few months.


Feedback:

- I've unchecked checkboxes that is clearly not done.
- You should mark `Firefox Add-ons SDK` as legacy so not to confuse users about what to use.
- It's Chrome, not chrome. API not Api. CRX, not crx.
- `Safari extensions API` => `Safari Extensions API`
- Safari also has app extensions now that are not mentioned: https://developer.apple.com/library/content/documentation/NetworkingInternetWeb/Conceptual/SafariAppExtension_PG/
- I would like to see more than just libraries. Include some blog posts, tutorials, links to community resources (Example: https://github.com/sindresorhus/awesome-electron#community), videos of talks, book, learning material, Stack Overflow tag, IRC, chat, forums, communities, etc.

**Comment by vitalets** [comment]
@sindresorhus thanks for feedback! Will do.

**Comment by vitalets** [comment]
@sindresorhus what do you think about adding special section *Pending Lists*? 
It's like incubator for growing lists that allows people to discover and contribute to it making list more community driven and grow faster. But in the same time it cleanly shows that list is not ready yet.

**Comment by davisonio** [comment]
Something similar to what you're describing has recently been added here; https://github.com/sindresorhus/awesome/wiki

**Comment by vitalets** [comment]
Great! Could I add https://github.com/vitalets/awesome-browser-extensions-and-apps there?

**Pull request closed without merging**